docs/uninstall.md

```diff
@@ -0,0 +1,21 @@
+# Uninstalling AzureAuth
+
+We provide an uninstall script you can use to Uninstall AzureAuth. The script removes the default AzureAuth reference in the PATH and it will deletes the AzureAuth installation folder.
+
+The script currently doesn't support uninstalling from custom locations. This is to avoid removing any potential files that are not safe to delete. If there are any potential installations in custom locations found in the PATH (installed through the `$AZUREAUTH_INSTALL_DIRECTORY` environment variable), a warning will be prompted showing its location. Installations in custom locations that are not listed in the PATH cannot be found and uninstalled.
+
+## Usage
+
+Run the following in PowerShell:
+
+```PowerShell
+$script = "${env:TEMP}\uninstall.ps1"
+$url = "https://raw.githubusercontent.com/AzureAD/microsoft-authentication-cli/main/install/uninstall.ps1"
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+Invoke-WebRequest $url -OutFile $script; if ($?) { &$script }; if ($?) { rm $script }
+```
+
+## Note
+
+* The script doesn't work for versions previous to AzureAuth 0.4.0

```

Comment:
Is this true? It deletes the whole default installation path.

Reply:
I just checked and it does work. A while ago I thought it wasn't going to work, I don't remember why tho. But I checked and it does get rid of the folder and the PATH reference.

Reply:
Removed the Note section, it's not true. It can remove those versions of AzureAuth